examples/complete_workflow/AuthoritationCodeCompleteWorkFlow.java

```diff
@@ -0,0 +1,157 @@
+package complete_workflow;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.CompletionException;
+
+import org.apache.hc.core5.http.ParseException;
+
+import com.neovisionaries.i18n.CountryCode;
+import com.wrapper.spotify.SpotifyApi;
+import com.wrapper.spotify.SpotifyHttpManager;
+import com.wrapper.spotify.exceptions.SpotifyWebApiException;
+import com.wrapper.spotify.model_objects.credentials.AuthorizationCodeCredentials;
+import com.wrapper.spotify.model_objects.credentials.ClientCredentials;
+import com.wrapper.spotify.model_objects.specification.AlbumSimplified;
+import com.wrapper.spotify.model_objects.specification.Artist;
+import com.wrapper.spotify.model_objects.specification.Paging;
+import com.wrapper.spotify.model_objects.specification.PagingCursorbased;
+import com.wrapper.spotify.model_objects.specification.PlayHistory;
+import com.wrapper.spotify.model_objects.specification.SavedAlbum;
+import com.wrapper.spotify.model_objects.specification.Track;
+import com.wrapper.spotify.model_objects.specification.User;
+import com.wrapper.spotify.requests.authorization.authorization_code.AuthorizationCodeRefreshRequest;
+import com.wrapper.spotify.requests.authorization.authorization_code.AuthorizationCodeRequest;
+import com.wrapper.spotify.requests.authorization.authorization_code.AuthorizationCodeUriRequest;
+import com.wrapper.spotify.requests.data.albums.GetAlbumRequest;
+import com.wrapper.spotify.requests.data.artists.GetArtistsTopTracksRequest;
+import com.wrapper.spotify.requests.data.artists.GetArtistsTopTracksRequest.Builder;
+import com.wrapper.spotify.requests.data.library.GetCurrentUsersSavedAlbumsRequest;
+import com.wrapper.spotify.requests.data.player.GetCurrentUsersRecentlyPlayedTracksRequest;
+import com.wrapper.spotify.requests.data.player.StartResumeUsersPlaybackRequest;
+import com.wrapper.spotify.requests.data.search.simplified.SearchAlbumsRequest;
+import com.wrapper.spotify.requests.data.search.simplified.SearchArtistsRequest;
+import com.wrapper.spotify.requests.data.search.simplified.SearchTracksRequest;
+import com.wrapper.spotify.requests.data.users_profile.GetCurrentUsersProfileRequest;
+
+public class AuthoritationCodeCompleteWorkFlow {
+	/*
+	* Before testing the API you must register your application on the Spotify Developers DashBoard. 
+	* After registering, copy your cient_id and secret_id and paste them into the following constants. 
+	* You will also need to have a redirect address, to which the user will be redirected after authorisation 
+	* to use the app, which must be authorised in the Spotify Developer DashBoard in the configuration area. 
+	* In the URI after the redirection there will be a code that will be used to exchange it for access and 
```

Comment:
Please change the wording so that it is clear what is meant by "In the URI after the redirection".